AirCasting/CreateSessionViews/ChooseSessionType/ChooseSessionTypeViewModel.swift

```diff
@@ -67,6 +67,10 @@ class ChooseSessionTypeViewModel: ObservableObject {
     }
     
     func syncButtonTapped() {
+        guard !userSettings.syncOnlyThroughWifi || networkChecker.isUsingWifi else {
+            alert = InAppAlerts.noWifiNetworkSyncAlert()
+            return
+        }
```

Comment:
don't we want this here too like in dashboard?:
```Swift
guard networkChecker.connectionAvailable else {
                 alert = InAppAlerts.noNetworkSyncAlert()
                 isRefreshing = false
                 return
             }
```

Reply:
It is present below (but as an if)